tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -244,27 +246,28 @@ export const uploadResource = async (args: uploadResourceArgs): Promise<void> =>
 
 export interface downloadResourcesArgs {
   page: Page
-  names: string[]
+  resources: [Object: any]
```

Comment:
Will this work? should avoid using any
```suggestion
interface someType {
      name: string
      type: string
}
...
  resources: someType[]
```